airbyte-integrations/connectors/source-mongodb-v2/src/main/resources/spec.json

```diff
@@ -98,13 +105,6 @@
         "default": "admin",
         "examples": ["admin"],
         "order": 4
-      },
```

Comment:
you will need to make additionalProperties=true for this to be backwards compatible, and potentially handle it manually in the connector code

Reply:
Thanks a lot, I've added.